### PR TITLE
build file debug, promise debug

### PR DESCRIPTION
**Pull request by arieljake**

### CODE REVIEW

**Comment by arieljake** [comment]
includes pull request from epegzz

addresses issues #34 and #35

File: www/lib/CordovaAppLoader.js [hunk 1]
```diff
@@ -172,8 +172,11 @@ var CordovaAppLoader =
 	            self.corruptNewManifest = true;
 	            resolve(null);
 	          }
-	          // Yes, we've updated and we've succeeded.
-	          resolve(false);
+		      else
```
Comment (micky2be) [inline www/lib/CordovaAppLoader.js]: Maybe it's my OCD kicking in but could you be careful with indentation

Comment (arieljake) [inline www/lib/CordovaAppLoader.js]: I have OCD about these issues as well...this is what I see on my machine:

image

Comment (micky2be) [inline www/lib/CordovaAppLoader.js]: Some identation are spaces others are tabs

**Pull request merged**